prio/prg.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2018, Henry Corrigan-Gibbs
+ * 
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. 
+ */
+
+#include <mprio.h>
+#include <nss/pk11pub.h>
+#include <strings.h>
+
+#include "prg.h"
+#include "rand.h"
+#include "share.h"
+#include "util.h"
+
+#define AES_BLOCK_SIZE 16
```

Comment:
You could take this from `blapit.h` instead of re-defining it here.

Reply:
Great. Will do.